pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -186,8 +186,8 @@ func (ku *KubeUtil) GetPodForContainerID(containerID string) (*Pod, error) {
 		return nil, err
 	}
 	pod, err := ku.searchPodForContainerID(pods, containerID)
-	if err != nil {
-		log.Debugf("cannot get the containerID %q: %s, retrying without cache...", containerID, err)
+	if err != nil && errors.IsNotFound(err) {
+		log.Debugf("Cannot get the containerID %q: %s, retrying without cache...", containerID, err)
```

Comment:
not sure this should be capitalized?

Reply:
It's mandatory to lowercase the errors but not the log messages. 
Me too, I'm not sure about the code style for that.